### ISSUE TITLE
Css fix

### ISSUE THREAD

**Issue by behu-kea** [Jacobkri93/PortfolieJacob#1]
` <link rel="stylesheet" type="text/css" href="styling/styles.css">`

Det her skulle fixe problemet. Alle ting derligger i static mappen skal bare refereres direkte fra static mappen

https://github.com/Jacobkri93/PortfolieJacob/blob/1dda84ee247dbd5f6c1c3a91b54ffc6c10128cc7/src/main/resources/templates/index.html#L7

**Comment by Jacobkri93** [comment]
Super, tak - det fiksede det faktisk, men nu kan jeg ikke få styling med andet end på 8080 og når jeg kører det i intelleJ. 

Selve linket fra settings viser det ikke her: https://jacobkri93.github.io/PortfolieJacob/

Har forsøgt både at lave rettelsen i intelleJ og pushe det, men det gjorde ikke noget.

Nogle ideer til hvad det kan skyldes?

**Comment by behu-kea** [comment]
Ja, det er fordi du kører projektet igennem spring boot. Github understøtter ikke backend server på den måde, men kun frontend (html, css og javascript)

Så du kan enten:

  1.  Kun tilføje html og css filer til dit portfolio
  2.  Lægge dit projekt på en service der understøtter server som heroku, aws etc.

Da dit portfolio nok er ret simpelt og der ikke er brug for en server vil jeg nok anbefale 1. Det er nok lidt overkill at spinne en springboot server op for et portfolio.

Hvis du vælger 2. så snak lige med Alexander Jens Peter Mavrina Sørensen. Han har deployet et projekt til heroku som virker.

Held og lykke med det 😊

From: Jacobkri93 ***@***.***>
Reply to: Jacobkri93/PortfolieJacob ***@***.***>
Date: Thursday, 25 March 2021 at 10.46
To: Jacobkri93/PortfolieJacob ***@***.***>
Cc: Benjamin Hughes ***@***.***>, Author ***@***.***>
Subject: Re: [Jacobkri93/PortfolieJacob] Css fix (#1)


Super, tak - det fiksede det faktisk, men nu kan jeg ikke få styling med andet end på 8080 og når jeg kører det i intelleJ.

Selve linket fra settings viser det ikke her: https://jacobkri93.github.io/PortfolieJacob/

Har forsøgt både at lave rettelsen i intelleJ og pushe det, men det gjorde ikke noget.

Nogle ideer til hvad det kan skyldes?
